ansible/README.md

```diff
@@ -0,0 +1,31 @@
+its configuration on all of your Mesos hosts can involve some repetitive busy work.
+We've provided some [Ansible](http://www.ansible.com/) Roles to help you automate these tasks.
+
+These Roles deploy satellite-master and satellite-slave.  systemd is used to launch and supervise the respective satellite processes.  Execute either Role to deploy Satellite initially, OR to apply your latest configuration to an existing installation of Satellite.
+
+
+## Instructions
+
+1. Create or adjust the Ansible inventory for your Mesos cluster.  You can just list hosts in a flat file (see inventory.sample), or build your inventory dynamically using the more [sophisticated capabilities of Ansible](http://docs.ansible.com/ansible/intro_dynamic_inventory.html).  Either way, you should arrange your inventory such that it contains the groups 'satellite\_masters' and 'satellite\_slaves', which contain the list of hosts to which you intend to deploy satellite\_master and satellite\_slave respectively.
+
+2. Copy vars.yml.sample to vars.yml, and edit it so that the file paths are appropriate for your workstation and your servers.  Note that some additional variables are specified in roles/satellite\_master/defaults/main.yml and roles/satellite\_slave/defaults/main.yml; you can override these if you need to.
+
+3. Edit the Satellite configuration files which you specified in your vars.yml (we've provided samples).
+
+4. Edit the Reimann configuration file at satellite-master/config/riemann-config.clj, if you'd like to customize the behavior of Riemann on your Satellite masters.
+
+5. cd to the directory containing this file.
+
+6. Run "ansible-playbook -i your_inventory example_playbook.yml" to deploy and start satellite\_master on your Mesos master servers, and to deploy and start satellite\_slave on your Mesos slave servers.
+
+
+## Want to use Supervisord (or another supervision solution) instead of Systemd?
+
+If you don't want to use systemd, when running the satellite\_master and satellite\_slave Roles, make sure to exclude the tasks that have the tag "systemd", using "--skip-tags systemd".
+
+This repository also includes some Ansible Roles which illustrate how you can supervise Satellite using Supervisord:  satellite\_master\_supervisor and satellite\_slave\_supervisor. There's also a [simple playbook](example_supervisord_playbook.yml) that runs these roles.
+
+These Supervisor Roles illustrate how you can deploy Satellite and make sure it stays running via Supervisor, but you probably won't be able to use the Roles on your cluster without modification.  The Roles will actually install Supervisor on your hosts;  assuming that you already have installed Supervisor, you'll want to skip the redundant Tasks.  You also will probably need to add additional variables to your Playbook so that Ansible's [supervisorctl module](http://docs.ansible.com/ansible/supervisorctl_module.html) can successfully communicate with your installation of Supervisor.
```

Comment:
@dgrnbrg Any or all of the variables exposed by Ansible's supervisorctl module might need to be overridden to a playbook in order to use a given installation of supervisorctl (location of the executable, username, password, etc).  I linked to the docs page for that Ansible module... if we enumerate all the variables it currently exposes in our own docs, we risk becoming out of date.  Maybe I should just be more explicit about why I'm linking to it?